web-build/Dockerfile.spx

```
@@ -0,0 +1,30 @@
+RUN <<EOS
+set -e
+git clone --depth 1 --branch $PHPSPX_VERSION_TAG https://github.com/NoiseByNorthwest/php-spx.git .
+# Ensure the tag represents what we think it should.
+git checkout $PHPSPX_COMMIT_HASH
+phpize${DDEV_PHP_VERSION}
+./configure --with-php-config=$(which php-config${DDEV_PHP_VERSION})
+make
+make install
+EOS
+
+WORKDIR /etc/php/$DDEV_PHP_VERSION/mods-available
```

Hadolint (2.12.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)